HISTORY.md

```diff
@@ -8,6 +8,7 @@
 * The EncryptionProvider::AddCipher method has been added to allow keys to be added to an EncryptionProvider.  This API will allow future providers to support multiple cipher keys.
 * Add a new option "allow_data_in_errors". When this new option is set by users, it allows users to opt-in to get error messages containing corrupted keys/values. Corrupt keys, values will be logged in the messages, logs, status etc. that will help users with the useful information regarding affected data. By default value of this option is set false to prevent users data to be exposed in the messages so currently, data will be redacted from logs, messages, status by default.
 * AdvancedColumnFamilyOptions::force_consistency_checks is now true by default, for more proactive DB corruption detection at virtually no cost (estimated two extra CPU cycles per million on a major production workload). Corruptions reported by these checks now mention "force_consistency_checks" in case a false positive corruption report is suspected and the option needs to be disabled (unlikely). Since existing column families have a saved setting for force_consistency_checks, only new column families will pick up the new default.
+* A new option, db_host_location, has been added to DBOptions. If left empty, this option defaults to the hostname (FQDN). Either the default or the user specified string is written to SST files as a table property
```

Comment:
Ok, we can do that. I'm not keen on introducing another option field to disable this though. How about default initialize ```options.db_host_location``` to some well known string, like "default" or "hostname"? We can parse it in ```SanitizeOptions``` and replace it with the actual hostname. To disable the property completely, the user can just set ```options.db_host_location``` to an empty string.

Reply:
Can it be something like default value to be "___automatic___"?

Reply:
How about this as a solution:
- We do not sanitize the DBOption but leave it "as is"
- If the "host_location" is empty, nothing is written to the SST file
- If the "host_location" is "localhost" or "default", the host name is written to the SST file (the Env is available in the options)

Also, it is not really a "location" but more of a "name" or "id".  

I still do not understand the use case for this location parameter and why I would want to use it.  What about the other table formats?

Reply:
Both sound like good ideas. Personally I like "__hostname__" or something like that as the "__" hints that it will be substituted with something, and "__hostname__" makes it clear what that something is. I think there's little risk of anyone taking it literally.

Regarding the option name, maybe "db_host_id" sounds better.

Reply:
> How about this as a solution:
> 
> * We do not sanitize the DBOption but leave it "as is"

That's a good point. That way the OPTIONS file can be read back in and validated without issues (which may have been your original concern).